packages/zero-protocol/src/protocol-version.test.ts

```diff
@@ -11,6 +11,6 @@ test('protocol version', () => {
   // If this test fails upstream or downstream schema has changed such that
   // old code will not understand the new schema, bump the
   // PROTOCOL_VERSION and update the expected values.
-  expect(hash).toEqual('2m9bqgxnw1vrj');
+  expect(hash).toEqual('7y9xjbvqi0sm');
```

Comment:
This technically changes the protocol, but `process-queries.ts` never passed back any errors until this PR and `ErroredQuery` was only a stubbed protocol change. So IIUC this doesn't necessitate a `PROTOCOL_VERSION` bump.